src/pkg/message/credentials.go

```diff
@@ -60,43 +62,56 @@ func PrintCredentialTable(state *types.ZarfState, componentsToDeploy []types.Dep
 }
 
 // PrintComponentCredential displays credentials for a single component
-func PrintComponentCredential(state *types.ZarfState, componentName string) {
+func PrintComponentCredential(ctx context.Context, state *types.ZarfState, componentName string) {
```

Comment:
See comments below in on `PrintCredentialUpdates()`. Recommend duplicating the control flow, adding logger in its place, and moving the function nearer to the cmd it's used in.